description.txt

```diff
@@ -1 +1 @@
-An in-game tool to inspect node parameters and metadata.
+An in-game tool to inspect node and item parameters and metadata.
```

Comment:
You may delete `depends.txt` and `description.txt`.
`remove_detached_inventory` was added in 5.0.0-dev, thus 0.4.x compatibility won't be needed.